survival_sponsorship_compassion/models/recurring_contract.py

```diff
@@ -0,0 +1,23 @@
+##############################################################################
+#
+#    Copyright (C) 2023 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Simon Gonzalez <sgonzalez@ikmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import fields, models, api
+from odoo.exceptions import UserError
+
+
+class SponsorshipContract(models.Model):
+    _inherit = ["recurring.contract"]
+    _name = "recurring.contract"
```

Comment:
This is not necessary when you do simple inheritance (name of the class is the same). You can change this everywhere.
```suggestion
    _inherit = "recurring.contract"
```